app/views/BookDetail/index.tsx

```diff
@@ -16,6 +23,35 @@ import Pooja from '../HomePage/temp-img/Pooja.png';
 import Footer from '#components/Footer';
 
 import styles from './styles.css';
+import { BACKEND_SERVER_URL } from '#base/configs/env';
+
+const BOOK_DETAIL = gql`
+query MyBookDetail ($id: ID!){
+    book(id: $id ) {
+        description
+        id
+        image
+        isbn
+        edition
+        language
+        price
+        metaDescription
+        metaKeywords
+        metaTitle
+        numberOfPages
+        ogDescription
+        ogImage
+        ogLocale
+        ogTitle
+        ogType
```

Comment:
what does og mean? Just curious

Reply:
Og is metadata for social media for (SEO)